nodes/persistence-node/src/createIpfsNode.ts

```diff
@@ -1,7 +1,26 @@
-import * as IPFS from 'ipfs-core'
+import { create as createInternalpfsNode, IPFS } from 'ipfs-core';
+import { create as createIpfsHttpClient } from 'ipfs-http-client'
+import { IpfsConfig } from './config/IpfsConfig';
+
+interface IDeps {
+  ipfsConfig: IpfsConfig
+}
+
+export const createIpfsNode = async (deps: IDeps): Promise<IPFS> => {
+  const ipfsUrl = deps.ipfsConfig.ipfsUrl
+
+  let ipfsNode: IPFS;
+
+  if (ipfsUrl) {
+    ipfsNode = await createIpfsHttpClient({
+      url: ipfsUrl,
+    });
+    console.log(`Using IPFS node at ${ipfsUrl}`);
+  } else {
+    ipfsNode = await createInternalpfsNode();
+    console.log("Using local IPFS node");
```

Comment:
Use logger, also if you see more console logs, feel free to replace with logger

Reply:
Not easily possible, as this whole function is ran during the initialization of DI container, before`logger.shouldLog` param is set, so `Logger` is not yet working at the time.